api/openapi.yaml

```diff
@@ -241,13 +282,61 @@ paths:
             application/json:
               schema:
                 $ref: ./model/GenericId.v1.yaml
+        '400':
+          description: The place already exists inside the current organization.
         '401':
           $ref: '#/components/responses/Unauthorized'
         '403':
           $ref: '#/components/responses/Forbidden'
+        '501':
+          $ref: '#/components/responses/NotImplemented'
+        default:
+          $ref: '#/components/responses/DefaultResponse'
+      summary: Add a new place.
+  /organization/{organizationId}/place/{placeId}:
+    parameters:
+      - $ref: '#/components/parameters/organizationId'
+      - $ref: '#/components/parameters/placeId'
+    put: #TODO
+      tags:
+        - stalker web app
+      description: Change data for a specific place. Operation reserved only to the managers and up.
+      operationId: putPlaceData
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: ./model/Place.v1.yaml
+      responses:
+        '200':
+          $ref: '#/components/responses/Ok'
+        '400':
+          $ref: '#/components/responses/NotFound'
+        '501':
+          $ref: '#/components/responses/NotImplemented'
+        default:
+          $ref: '#/components/responses/DefaultResponse'
+      summary: Update a place.
+    delete: #TODO
+      tags:
+        - stalker web app
+      description: Delete the specified place. Operation reserved only to the managers and up.
```

Comment:
Siamo sicuri di volere una delete? Non c'era il discorso che con una delete le chiamate non sono idempotenti?

Reply:
a questa ti rispondo io: la lasciamo come delete per ora e più avanti se ci avanza tempo rifattorizziamo tutte le DELETE trasformandole in una post (?) al sotto-endpoint .../delete